docs/docs/hello-world.md

```diff
@@ -3,6 +3,25 @@ id: hello-world
 title: Hello World
 permalink: docs/hello-world.html
 prev: installation.html
+next: introducing-jsx.html
 ---
 
-TODO
+The smallest React example looks like this:
+
+```js
+let element = <h1>Hello, world!</h1>;
+let domNode = document.getElementById('root');
+ReactDOM.render(element, domNode);
+```
+
+**[Try it on Codepen.](http://codepen.io/gaearon/pen/ZpvBNJ?editors=0010)**
+
+It renders a header saying "Hello World" on the page.
+
+There are a few things to unpack here:
```

Comment:
This is exactly what I plan to do, from simple to more complex. I didn't want to introduce too many concepts right away so I wanted to keep Hello World super short.

In future topics, you see how JSX interpolation works ("JSX"), how React preserves DOM despite re-renders ("Elements" section), how to encapsulate reusable parts of UI ("components as props"), and how to make things change on the screen ("events, lifecyle, and state").

I didn't want to start with components because this brings too many unknowns at once. For people comfortable with that, there is a tutorial, but I want to also have a path that offers full gradual understanding of public API.